scripts/core/editor3/components/handlePastedText.js

```diff
@@ -121,22 +123,20 @@ function processPastedHtml(props, html) {
     );
 
     const selectionAfterInsert = nextEditorState.getSelection();
-
-    nextEditorState = EditorState.set(nextEditorState, {allowUndo: false});
```

Comment:
Does undo/redo work properly after removing this?

Reply:
yes